roles/kvirt_vm/README.md

```diff
@@ -0,0 +1,106 @@
+# kvirt_vm role
+
+This role allows the deployment of a Kubevirt virtual machine. In order to execute the role a running OpenShift cluster and its credentials are required. i.e. through the KUBECONFIG environment variable.
+
+The role is aimed for deployment if virtual nodes for OCP deployments, so only the root disk is created.
```

Comment:
Here, maybe was `deployment of virtual nodes` instead of `deployment if virtual nodes` ?